lib/jsonld.js

```diff
@@ -812,7 +812,7 @@ jsonld.merge = util.callbackify(async function(docs, ctx, options) {
   // compact result (force @graph option to true, skip expansion)
   options.graph = true;
   options.skipExpansion = true;
-  const compacted = jsonld.compact(flattened, ctx, options);
+  const compacted = await jsonld.compact(flattened, ctx, options);
```

Comment:
My concern here is - why didn't the `merge()` unit test in `misc.js` catch this line?

Reply:
The tests don't pass in a context, so the flattened value is returned a few lines up and this code is never run.